content/docs/04.5-devx/07-sandbox-clusters/00-pause-restore-virtual-clusters.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Pause and Restore Palette Virtual Clusters"
+metaTitle: "Palette Dev Engine Pause and Restore Palette Virtual Clusters"
```

Comment:
If you change the title, does the metaTitle also need updating to match?

Reply:
not mandatory